packages/admin/admin-stories/src/admin/form/RangeInput.tsx

```diff
@@ -1,5 +1,6 @@
 import { Field, FinalFormRangeInput, Toolbar, ToolbarTitleItem } from "@comet/admin";
-import { Box, Button, Card, CardContent, styled, Typography } from "@mui/material";
+import { Box, Button, Card, CardContent, Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
```

Comment:
Why do we need to change the import? Shouldn't this work when using `styled` from `@mui/material`?

Reply:
because mui says so: https://mui.com/system/styled/#import-path
this has to be done now for all projects